Formula/vgrep.rb

```diff
@@ -1,8 +1,8 @@
 class Vgrep < Formula
   desc "User-friendly pager for grep"
   homepage "https://github.com/vrothberg/vgrep"
-  url "https://github.com/vrothberg/vgrep/archive/v5.2.2.tar.gz"
-  sha256 "5132ef6b254bfb8535b4021c297aaeafa1e641de5ab3d1ba0e1748586f97d192"
+  url "https://github.com/vrothberg/vgrep/archive/v2.5.3.tar.gz"
+  sha256 "d2fa1c1ac7e196d591b834cccff1cb457d20e66d03558e7f7b3b966bef3ab86c"
   license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-only"
  version_scheme 1
```
This should fix the audit complaining about the version decrease.

Reply:
Done :)